lein-eftest/src/leiningen/eftest.clj

```diff
@@ -60,18 +68,30 @@
                           ~selectors)]
       ns#)))
 
+(defn- process-options
+  [{:keys [report report-to-file] :as options}]
+  ;; Use `or` as opposed to on the keyword's IFn to ensure we override an
+  ;; explicit/accidental `nil`.
```

Comment:
This comment is wonky. I think I meant "Use `or` as opposed to relying on the keyword's IFn". I'll have to fix that.